docs/monitoring-usage.rst

```diff
@@ -278,3 +278,110 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Time Series
+---------------------------
+
+The Stackdriver Monitoring API can be used to create and delete custom metrics, as well as write
```

Comment:
It can be used to create and delete _metric descriptors_ for custom metrics, but that's already discussed earlier in the document and doesn't need to be repeated here.